plugin/src/main/java/org/opensearch/ml/action/model_group/TransportUpdateModelGroupAction.java

```diff
@@ -186,6 +187,7 @@ private void updateModelGroup(
 
     private void updateModelGroup(String modelGroupId, Map<String, Object> source, ActionListener<MLUpdateModelGroupResponse> listener) {
         UpdateRequest updateModelGroupRequest = new UpdateRequest();
+        updateModelGroupRequest.setRefreshPolicy(WriteRequest.RefreshPolicy.IMMEDIATE);
```

Comment:
Just curious why only in update model group? Why not for create & delete operations?